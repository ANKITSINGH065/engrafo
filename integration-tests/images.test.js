var utils = require("./utils");

test("images in figures render correctly", done => {
  utils.expectBodyToMatchSnapshot("images/figure.tex", done);
});
