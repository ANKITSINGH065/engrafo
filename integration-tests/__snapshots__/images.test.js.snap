// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`images in figures render correctly 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    

    <dt-article>
      

      <h1>
        Untitled
      </h1>
      

      <div
        class="engrafo-metadata"
      >
        
  
        <div
          class="authors"
        >
          
    
        </div>
        
  
        <div
          class="engrafo-metadata-custom"
        />
        

      </div>
      

      <figure
        id="figure-1"
      >
        

        <img
          alt="Figure 1: Fig."
          src="fig.png"
          title="fig:"
        />
        <figcaption>
          Figure 1: Fig.
        </figcaption>
        

      </figure>
      

      <p>
        See 
        <a
          href="#figure-1"
        >
          Figure 1
        </a>
        .
      </p>
      

      <p>
        Ref which doesnâ€™t exist  
        <span
          class="engrafo-missing-ref"
        >
          ?
        </span>
        .
      </p>
      

    </dt-article>
    

    <dt-appendix>
      

      


      <div
        class="l-body"
      />
      

    </dt-appendix>
    



    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;

exports[`multiple images in figures render correctly 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    

    <dt-article>
      

      <h1>
        Untitled
      </h1>
      

      <div
        class="engrafo-metadata"
      >
        
  
        <div
          class="authors"
        >
          
    
        </div>
        
  
        <div
          class="engrafo-metadata-custom"
        />
        

      </div>
      

      <figure
        class="l-page"
        id="figure-1"
      >
        

        <div
          class="images"
        >
          <div>
            <img
              alt="Figure 1: Fig."
              src="fig.png"
              title="fig:"
            />
          </div>
          <div>
            <img
              alt="Fig."
              src="fig2.png"
              title="fig:"
            />
          </div>
        </div>
        <figcaption>
          Figure 1: Fig.
        </figcaption>
        

      </figure>
      

      <p>
        See 
        <a
          href="#figure-1"
        >
          Figure 1
        </a>
        .
      </p>
      

    </dt-article>
    

    <dt-appendix>
      

      


      <div
        class="l-body"
      />
      

    </dt-appendix>
    



    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;

exports[`plain images render correctly 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    

    <dt-article>
      

      <h1>
        Untitled
      </h1>
      

      <div
        class="engrafo-metadata"
      >
        
  
        <div
          class="authors"
        >
          
    
        </div>
        
  
        <div
          class="engrafo-metadata-custom"
        />
        

      </div>
      

      <figure
        id=""
      >
        

        <img
          alt="image"
          src="fig.png"
        />
        

      </figure>
      

    </dt-article>
    

    <dt-appendix>
      

      


      <div
        class="l-body"
      />
      

    </dt-appendix>
    



    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;
